catalog/app/components/JsonDisplay/JsonDisplay.js

```diff
@@ -336,7 +322,7 @@ export default function JsonDisplay({
 }) {
   const ref = React.useRef(null)
   const classes = useStyles()
-  const currentBPWidth = useElementWidth(ref)
+  const { width: currentBPWidth } = useResizeObserver({ ref })
   const computedKeys = React.useMemo(() => {
     if (showKeysWhenCollapsed === true) return Number.POSITIVE_INFINITY
     if (showKeysWhenCollapsed === false) return Number.POSITIVE_INFINITY
```

Comment:
This looks awkward. We return the same thing not matter the state of `showKeysWhenCollapsed`?

Reply:
Yes, probably, should be NEGATIVE_INFINITY or `0`. The code was never changed since created https://github.com/quiltdata/quilt/pull/1806, but, this condition was not used when it was added, and it is not used today :)
I'll remove it in standalone PR